stubs/peewee/METADATA.toml

```diff
@@ -0,0 +1,2 @@
+version = "3.14.10.*"
+python2 = true
```

Comment:
I added this because it looked like you were trying to make the stub Python 2-compatible, but note that we're planning on phasing out support for Python 2 soon (see #7367)